chain/chain/src/chain.rs

```diff
@@ -3977,6 +3956,11 @@ impl Chain {
         let shard_uid = self.epoch_manager.shard_id_to_uid(shard_id, block.header().epoch_id())?;
         let is_new_chunk = chunk_header.height_included() == block.header().height();
         let shard_update_reason = if should_apply_transactions {
+            let block_context = self.get_apply_transactions_block_context(
```

Comment:
is var name accurate enough?

Reply:
🤔 do you have any suggestions for a more accurate name?

Reply:
Just an idea - could `RuntimeBlockContext` work?
It has less words and doesn't spoil definition much because transactions are applied on runtime.

Reply:
Yeah, I've considered `RuntimeBlockContext` but then I decided to go with something more specific because some method (for example `prepare_transactions`) do not require all that fields.
But I don't mind `RuntimeBlockContext` if you have a strong preference.